middleware_context_vars.go

```diff
@@ -62,6 +63,9 @@ func (m *MiddlewareContextVars) ProcessRequest(w http.ResponseWriter, r *http.Re
 	// IP:Port
 	contextDataObject["remote_addr"] = copiedRequest.RemoteAddr
 
+	//Correlation ID
+	contextDataObject["cid"] = uuid.NewV4().String()
```

Comment:
`cid` doesn't seem as descriptive on its own as others like `remote_addr`. Perhaps `corr_id`? Or perhaps name it something like "request id"?